ScorchApp/src/plugins/notify.js

```diff
@@ -0,0 +1,52 @@
+import { NotificationAlert } from 'components/util'
+
+const Notify = {
+  install (Vue, options) {
+    let baseNotification = function(msg){
+      document.getElementById("notification-msg").innerHTML = msg;
+      document.getElementById("notification-container").classList.add('notify-active'); 
+      document.getElementById("notification-container").classList.add('visible');
+      document.getElementById("notification-container").classList.remove('invisible');
+    }
+
+    let clearClasses = function(){
+      setTimeout(() => {
+        document.getElementById("notification-container").classList.add('invisible');
+        document.getElementById("notification-container").classList.remove('visible');
+        document.getElementById("notification-container").classList.remove('alert-info');
+        document.getElementById("notification-container").classList.remove('alert-warning');
+        document.getElementById("notification-container").classList.remove('alert-danger');
+        document.getElementById("notification-container").classList.remove('alert-success');
+        document.getElementById("notification-container").classList.remove('notify-active'); 
+      }, 3950);
+    }
+
```

Comment:
instead of adding so many prototypes, we should consolidate to one, this.$notify, so we know they are all part of the same plugin